lib/main.coffee

```diff
@@ -43,12 +43,12 @@ module.exports =
       name: 'coala'
       grammarScopes: ['*']
       scope: 'file'
-      lintOnFly: true
+      lintsOnChange: true
       lint: (textEditor) =>
         filePath = textEditor.getPath()
         parameters = []
         parameters.push '--find-config'
-        parameters.push '--limit-files=' + filePath
+        parameters.push '--limit-files=' + path.basename(filePath)
```

Comment:
the basename is usually the part after the last slash right? So it's not necessarily relative to the project directory.